packages/did-provider-ebsi/src/ebsi-did-provider.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/require-await */
+import { IAgentContext, ICredentialPlugin, IIdentifier, IKey, IKeyManager, IService } from '@veramo/core'
+import { base58btc } from 'multiformats/bases/base58'
+import { randomBytes } from 'crypto'
+import * as jose from 'jose'
+import { AbstractIdentifierProvider } from '@veramo/did-manager'
+import { onboard } from './ebsi-did-onboarding.js'
+import { privateKeyJwkToHex } from './ebsi-did-utils.js'
+
+type IContext = IAgentContext<IKeyManager | ICredentialPlugin>
+
+/**
+ * {@link @veramo/did-manager#DIDManager} identifier provider for `did:ebsi` identifiers
+ *
+ * @beta This API may change without a BREAKING CHANGE notice.
+ */
+export class EbsiDIDProvider extends AbstractIdentifierProvider {
+  private defaultKms: string
+
+  constructor(options: { defaultKms: string }) {
+    super()
+    this.defaultKms = options.defaultKms
+  }
+
+  async createIdentifier(
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    { kms, options }: { kms?: string; options?: any },
+    context: IContext,
+  ): Promise<Omit<IIdentifier, 'provider'>> {
+    if (!options.bearer) {
+      throw new Error('Bearer token should be provided in options argument')
+    }
+    const bearer = options.bearer as string
+
+    const keys = await jose.generateKeyPair('ES256K')
```

Comment:
https://github.com/uport-project/veramo/blob/next/packages/did-provider-jwk/src/jwk-did-provider.ts#L31

Can you add the option to import keys instead of always generating new ones?